### PR TITLE
[Doctrine] Update multiple_entity_managers.rst

### PR DESCRIPTION
**Pull request by noniagriconomie**
Document the way to bind this second entity manager so that it is injectable via typehint binding

### CODE REVIEW

**Comment by maxhelias** [comment]
FYI in the next DoctrineBundle 2.3 : https://github.com/doctrine/DoctrineBundle/pull/1259

File: doctrine/multiple_entity_managers.rst [hunk 1]
```diff
@@ -254,6 +254,17 @@ the default entity manager (i.e. ``default``) is returned::
...
 
+You can leverage the service binding feature to reference the ``customer``
+entity manager so that you can inject it directly::
+
+    services:
+        _defaults:
+            bind:
+                $customerEntityManager: '@doctrine.orm.customer_entity_manager'
+
+Now type-hint your method with `EntityManagerInterface $customerEntityManager`
```
Comment (OskarStark) [inline doctrine/multiple_entity_managers.rst]:
```suggestion
Now type-hint your method with ``EntityManagerInterface $customerEntityManager``
```

Comment (noniagriconomie) [reply doctrine/multiple_entity_managers.rst]: nice catch

**Comment by HypeMC** [comment]
FYI, DoctrineBundle 2.3 [is out](https://github.com/doctrine/DoctrineBundle/releases/tag/2.3.0), also there was a followup[ PR for connections](https://github.com/doctrine/DoctrineBundle/pull/1296), maybe it's worth adding that to the docs as well.

**Comment by javiereguiluz** [comment]
We finally merged this! Thanks Antoine.

**Pull request merged**